specs/core/0_beacon-chain.md

```diff
@@ -1563,10 +1515,10 @@ Verify that `len(block.body.exits) <= MAX_EXITS`.
 For each `exit` in `block.body.exits`:
 
 * Let `validator = state.validator_registry[exit.validator_index]`.
-* Verify that `validator.status == ACTIVE`.
+* Verify that `validator.exit_slot > state.slot + ENTRY_EXIT_DELAY`.
```

Comment:
Should be `== FAR_EXIT_SLOT`, otherwise the validator could initialize multiple exits as long as they do so before `ENTRY_EXIT_DELAY`, or if they got penalized, they could keep pushing back their exit time!